code/__DEFINES/traits.dm

```diff
@@ -127,10 +127,30 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 */
 
 //mob traits
+/// Forces the user to stay unconscious.
```

Comment:
Can we explicitly comment the reason for this existing, and that people should query stat but use this to make someone unconcious from a specific source. (This shouldn't be checked anywhere outside of update_stat, anywhere outside of update_stat should be querying stat instead.)

Reply:
Ill split them off, yeah

Reply:
split it off the rest of the mob traits and added more substantial comments. Added TODO to show my intentions.

4fc03d8